msg/ZFS-8000-ER/index.html

```diff
@@ -225,4 +225,91 @@
 all pools are healthy
 </pre>
 </td></tr></table>
+<br><br>
+<hr>
+<br><br>
+<!-- end: templates/success -->
+<!-- start: article -->
+<!-- code: ZFS-8000-ER -->
+<!-- keys: errata -->
+<table width="80%"><tr><td>
+<a name="4"><b>ZFS Errata #4</b></a>
+<p>
+<dl>
+<dt><p><b>Type</b>
+<dd>Compatibility
+<dt><p><b>Severity</b>
+<dd>Moderate
+<dt><p><b>Description</b>
+<dd>An encrypted dataset contains an on-disk format incompatibility.
+<dt><p><b>Automated Response</b>
+<dd>No automated response will be taken.
+<dt><p><b>Impact</b>
+<dd>
+Encrypted datasets created before the ZFS packages were updated cannot be
+backed up via a raw send to an updated system. These datasets also cannot
+receive additional snapshots.
+</dd>
+<dt><p><b>Suggested Action for System Administrator</b>
+<dd>
+First, system administrators with affected pools will need to enable the
+<b>bookmark_v2</b> feature on their pools. This will disable read-write
+and read-only compatibility with previous versions of the ZFS software.
+Next, administrators will need to backup all existing encrypted datasets.
+This can be done in several ways. Non-raw <b>zfs send</b> and
+<b>zfs receive</b> can be used as per usual, as can traditional backup
+tools.
+
+If the administrator wishes to continue using raw send streams, the
+zfs_disable_ivset_guid_check tunable must be set to 1 on the receive side.
+Streams received this way (as well as any received before the upgrade)
+will need to be checked to ensure they are not corrupted.
```

Comment:
checked how?  If we don't know, perhaps at least `manually checked`, to indicate that this isn't something that ZFS is going to do for them.